### ISSUE TITLE
dist.rpminspect.comparison.upstream says version did not change, but it did

### ISSUE THREAD

**Issue by thrix** [rpminspect/rpminspect#96]
The test dist.rpminspect.comparison.upstream, when comparing:
```
rh-jmc-jmc-core-7.1.0-1.el7 to rh-jmc-jmc-core-7.0.0-4.20190211hg62c5279452a6.el7
```
says
```
Unexpected changed source archive content. The version of the package did not change between builds, but the source archive content did. This may be deliberate, but needs inspection.

```
Which seems to be incorrect, the version changed :)

https://dashboard.osci.redhat.com/#/artifact/brew-build/aid/26352596?focus=id:cfef50e97308-2

**Comment by dcantrell** [comment]
You're right.  The package version comparison to determine if the result was INFO or VERIFY was only happened for changed sources, not removed or new.  I've changed up the way the inspection works now so if the package version changes, all upstream source changes are reported as INFO.  If the versions remain the same and things change, it becomes a VERIFY with the existing remedy text returned.
